check_manifest.py

```diff
@@ -1016,9 +1021,13 @@ def main():
     if args.ignore_bad_ideas:
         IGNORE_BAD_IDEAS.extend(args.ignore_bad_ideas.split(','))
 
-    if args.verbose:
+    verbosity = args.quiet + args.verbose
+    if verbosity >= 2:
         global VERBOSE
         VERBOSE = True
+    if not verbosity:
+        global QUIET
+        QUIET = True
```

Comment:
It would probably be nicer to have one global `VERBOSITY = 1` instead of juggling two variables, but eh, that can wait for some future refactoring.